sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationClientBuilderTest.java

```diff
@@ -97,85 +81,6 @@ public void nullAADCredential() {
         });
     }
 
```

Comment:
Is there a reason we removed these test cases? You can still test it, but you wouldn't be generating an actual client.

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/6675/files/ede705f56a3d69c3eed3765ea1769556f44874e2#diff-2705d6620e99b076f6e0a81319031b5bR671

Abstract the test into Client test base.